utils/build_system/create_test_release_jobs.py

```
@@ -205,6 +205,100 @@ def get_job_description(job_path: Path) -> str:
+
+        Returns:
+            True if the job was disabled, False if it doesn't exist or failed.
+        """
+        full_path = f"{self.base_job_dir}/{job_name}" if not job_name.startswith(self.base_job_dir) else job_name
+        try:
+            if not self.jenkins.job_exists(full_path):
+                LOGGER.warning("Job '%s' does not exist, skipping disable", full_path)
+                return False
+
+            xml_data = self.jenkins.get_job_config(full_path)
+            config = ET.fromstring(xml_data)
```

Ruff (0.15.14)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)